.claude/PHASE3_INTEGRATION_TEST.md

```
@@ -0,0 +1,307 @@
+---
+
+## 테스트 환경
+
+### Backend Server
+- **URL**: http://localhost:8080
+- **Profile**: local
+- **Database**: H2 in-memory
+- **APIs**:
+  - Spoonacular API (음식 데이터)
+  - Google Cloud Translation API (번역)
+  - Google Cloud Vision API (OCR)
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)